### PR TITLE
Rel 518437 misc rsapi cleanup

### PR DESCRIPTION
**Pull request by TommyBigNose**
- Updated README
- Renamed Document to DocumentHelper
- Cleaned up Test Example project
- Removed ForceDbContext

### CODE REVIEW

**Review by aarongilbert123 — APPROVED** [review]

**Review by aarongilbert123 — APPROVED** [review]

**Review by calimetikcura — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -1,69 +1,59 @@
...
-# Utilizing MailHelper for tests
-## Gmail
-With gmail, you will need to allow IMAP on your account.  It is recommended to make an alternate fake account just for testing.  With that, you will just need to use your gmail and gmail password in the GmailMailHelper constructor.
-## MailTrap
-[MailTrap Site]([https://mailtrap.io/inboxes](https://mailtrap.io/inboxes))
-You will need at least a free account with MailTrap to utilize these helpers.  The free account grants you a username, password, and Api Key.  The API Key is used to access your inbox which MailTrapMailHelper takes in from its constructor.
-## Licensing
-[MailKit NuGet](https://github.com/jstedfast/MailKit)
 
+This framework is only compatible for Relativity 9.5 and above.
```
[the remark this answers is not in the document]
Comment (TommyBigNose) [reply README.md:4]: I'll reword this to be more specific
